src/pages/CommitDetailPage/CommitDetailPageContent/CommitDetailPageContent.jsx

```diff
@@ -44,6 +48,8 @@ function CommitDetailPageContent() {
     return <ErroredUploads erroredUploads={erroredUploads} />
   }
 
+  const showIndirectChanges =
```

Comment:
Nicee, yeah I like this better than `hideIndirectChanges` and then doing `hidIndirectChanges ? null : <Component />`